README.md

```diff
@@ -65,6 +66,7 @@ Clone or copy the source into your own project. There are sections marked with `
 * testing
   * unit testing including mocking dependencies - unit testing of formExampleController with service mocked using mokito
   * browser testing - using [webdriver.io](http://webdriver.io/) to create browser tests, the configuration can be extended to call into remote selenium grids and services like [Saucelabs](https://saucelabs.com/) and [BrowserStack](https://www.browserstack.com/)
+  * accessibility testing - using [Pa11y](http://pa11y.org/) to test supplied URLs and generate HTML reports for any failures
```

Comment:
can you include information here about WAVE and how it can be used to manually test